docs/static/releasenotes.asciidoc

```diff
@@ -86,6 +87,18 @@ This section summarizes the changes in the following releases:
 * <<logstash-7-0-0-alpha1,Logstash 7.0.0-alpha1>>
 
 
+[[logstash-7-17-27]]
+=== Logstash 7.17.27 Release Notes
+
+No user-facing changes in Logstash core.
+
+==== Plugins
```

Comment:
```suggestion
[[logstash-plugins-7-17-27]]
==== Plugins
```

Reply:
I'm adding a new anchor to make conflict resolution easier when we release multiple versions.